lib/matplotlib/dviread.py

```diff
@@ -1017,16 +1017,16 @@ def find_tex_file(filename, format=None):
     if format is not None:
         cmd += ['--format=' + format]
     cmd += [filename]
-    try:  # Below: strip final newline.
-        result = cbook._check_and_log_subprocess(cmd, _log)[:-1]
```

Comment:
Why did this change?  

This is a little hard to review in the absence of tests;  What test should I run locally to make sure this doesn't break?

Reply:
Maybe simply replace `[:-1]` with `.rstrip('\r\n')`?

Reply:
That's what we do below...  I don't mind doing it for both windows and non-windows, it won't matter in practice.
The main difference is with the encoding though.

Reply:
@jklymak I do not know why old `TEST_ALL: "yes"` builds were removed without making at least a single newer one. I have started a `TEST_ALL: "yes"` Appveyor build https://ci.appveyor.com/project/Kojoley/matplotlib/build/1.0.133 in my fork to see how long it takes.